pkg/ddl/index.go

```diff
@@ -2217,6 +2217,9 @@ func (w *worker) executeDistTask(t table.Table, reorgInfo *reorgInfo) error {
 	return err
 }
 
+// EstimateTableRowSizeForTest is used for test.
+var EstimateTableRowSizeForTest = estimateTableRowSize
```

Comment:
Could we put it to `pkg/ddl/ddl_test.go`?

Reply:
No, we need to access it in `realtikvtest` package.